### ISSUE TITLE
Topiary crash

### ISSUE THREAD

**Issue by ihodes** [hammerlab/biokepi#440]
```
Traceback (most recent call last):
  File "/pgv-results/biokepi-coco-work/toolkit-coco/python-tools/envs/topiary.0.1.2/bin/topiary", line 33, in <module>
    from topiary.commandline_args import (
  File "/pgv-results/biokepi-coco-work/toolkit-coco/python-tools/envs/topiary.0.1.2/lib/python3.6/site-packages/topiary/__init__.py", line 7, in <module>
    from .predict_epitopes import (
  File "/pgv-results/biokepi-coco-work/toolkit-coco/python-tools/envs/topiary.0.1.2/lib/python3.6/site-packages/topiary/predict_epitopes.py", line 18, in <module>
    from mhctools import EpitopeCollection
ImportError: cannot import name 'EpitopeCollection'
```

**Comment by smondet** [comment]
Has this been reported upstream?

Or is it specific to our Conda ~hacks~ setup? ;)

**Comment by ihodes** [comment]
Reported upstream now, also /ccing @armish

**Comment by julia326** [comment]
Merged #449 (increasing topiary version) so this no longer fails.
